app/kuma-dp/pkg/dataplane/envoy/envoy.go

```diff
@@ -156,3 +167,27 @@ func (e *Envoy) Start(stop <-chan struct{}) error {
 		return err
 	}
 }
+
+func (e *Envoy) version() (*EnvoyVersion, error) {
+	binaryPathConfig := e.opts.Config.DataplaneRuntime.BinaryPath
+	resolvedPath, err := lookupEnvoyPath(binaryPathConfig)
+	if err != nil {
+		return nil, err
+	}
+	arg := "--version"
+	command := exec.Command(resolvedPath, arg)
+	runLog.Info("getting version of Envoy")
```

Comment:
I think this log is unnecessary. If everything works we will see `fetched envoy version` if it fails we will see `failed to get Envoy version`

Reply:
Ok, I removed this log.